Verse/src/Schemas/JSON/DecoderConverter.cs

```diff
@@ -28,15 +28,15 @@ internal class DecoderConverter : IDecoderConverter<JSONValue>
 
 		public Converter<JSONValue, TTo> Get<TTo>()
 		{
-		    if (!this.converters.TryGetValue(typeof (TTo), out var box))
+			if (!this.converters.TryGetValue(typeof (TTo), out var box))
 				throw new InvalidCastException(string.Format(CultureInfo.InvariantCulture, "cannot convert JSON value into '{0}', please register a converter using schema's SetDecoderConverter method", typeof (TTo)));
 
 			return (Converter<JSONValue, TTo>)box;
 		}
 
 		public void Set<TTo>(Converter<JSONValue, TTo> converter)
 		{
-            this.converters[typeof (TTo)] = converter ?? throw new ArgumentNullException(nameof(converter));
+			this.converters[typeof (TTo)] = converter ?? throw new ArgumentNullException(nameof(converter));

```

Comment:
[NIT] You have a few unwanted space/tab replacement in the review, could you please try not to change spacing?

Reply:
removed spacing changes